README.md

```diff
@@ -46,21 +46,33 @@ features = ["libm"] # Makes it use libm instead of std for float math
 
 ## It's Never "Just RGB"
 
-Colors in, for example, images are often "gamma corrected" or stored in sRGB format as a compression method and to prevent banding. This is also a bit of a legacy from the ages of the CRT monitors, where the output from the electron gun was nonlinear. The problem is that these formats don't represent the actual intensities, and the compression has to be reverted to make sure that any operations on the colors are accurate. This library uses a completely linear work flow, and comes with the tools for transitioning between linear and non-linear RGB.
+Colors in, for example, images, are often "gamma corrected", or converted using some non-linear transfer function into a format like sRGB before being stored or displayed. This is done as a compression method and to prevent banding, and is also a bit of a legacy from the ages of the CRT monitors, where the output from the electron gun was nonlinear. The problem is that these formats are *non-linear color spaces*, which means that many operations that you may want to perform on colors (addition, subtraction, multiplication, linear interpolation, etc.) will not work properly. As such, the compression has to be reverted to restore linearity and make sure that many operations on the colors are accurate. 
 
-Adding to that, there are more than one kind of non-linear RGB. Ironically enough, this turns RGB into one of the most complex color spaces.
+But, even when colors *are* 'linear', there is yet more to explore.
+
+The most common way that color spaces are defined, especially for computer storage, is in terms of so-called *tristimulus values*, meaning that all colors are defined as a vector of three values. The reason colors can generally be stored as only a three dimensional vector, and not an *n* dimensional one, where *n* is some number of possible frequencies of light, is because our eyes contain only three types of cones, each of which have different sensitivity curves to different wavelengths of light. These cones are often called the S, M, and L (for small, medium, and large) cones, and their sensitivity curves *roughly* position them as most sensitive to "red", "green", and "blue" parts of the spectrum, which is where the common 'RGB' concept comes from. And, as such, we can choose only three values to represent any possible color that a human is able to see. An interesting consequence of this is that humans can two different things which are emitting *completely different actual spectra* as the *same visual color* so long as those spectra, when added and transformed by the sensitivity curves of our cones, result in the same S, M, and L values which get transmitted to our brain.
```

Comment:
Looks like you accidentally a word or two in "An interesting consequence of this is that humans can two different things which are emitting"